src/components/InlineWrapper/InlineWrapper.scss

```diff
@@ -0,0 +1,22 @@
+@import '~@quartz/styles/scss/borders';
+@import '~@quartz/styles/scss/media-queries';
+@import '~@quartz/styles/scss/fonts';
+
+.container {
+	@include font-maison-500-3;
+
+	background-color: $color-background-2;
+	border: $border-solid-decorative;
+	border-radius: $size-border-radius;
+	margin: 20px 0 40px 0;
+	padding: 24px 12px;
+
+	@include for-tablet-portrait-up {
+		margin: 40px 0;
+		padding: 28px;
+	}
+
+	@media print {
+		display: none;
+	}
```

Comment:
```suggestion
```
I think this is right in some cases, e.g. article callouts, but probably won't be universally applicable

Reply:
👍